bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -817,72 +819,75 @@ integration.define({
         {
```

Comment:
You are missing string length for all parameters. Please put those back.

Reply:
![image (2)](https://user-images.githubusercontent.com/80905773/126978244-95ddda6c-5502-4505-a69d-cb278817e0e2.png)

when we use length property in action parameters we get the above error

Reply:
Oh, it's action, sorry I missed that. It's correct then, sorry.